src/bif/erlang.rs

```diff
@@ -0,0 +1,519 @@
+use crate::bif::BifResult;
+use crate::process::RcProcess;
+use crate::value::{self, Term, TryInto, Tuple};
+use crate::exception::{Exception, Reason};
+use crate::vm;
+
+pub fn bif_erlang_make_tuple_2(_vm: &vm::Machine, process: &RcProcess, args: &[Term]) -> BifResult {
+    let num = match args[0].into_number() {
+        Ok(value::Num::Integer(i)) if ! i < 0 => i,
+        _ => return Err(Exception::new(Reason::EXC_BADARG))
+    };
+    let heap = &process.context_mut().heap;
+    let tuple = value::tuple(&heap, num as u32);
+    for i in 0..num {
+        unsafe {
+            std::ptr::write(&mut tuple[i as usize], args[1].clone());
```

Comment:
No clone, Term is now Copy so we almost never need clone.

Make sure to run `cargo clean -p enigma && cargo clippy -- -W clippy::nursery -W clippy::all`, it'll show lint warnings for these types of changes